metaflow/plugins/kfp/s3_sensor_decorator.py

```diff
@@ -68,24 +68,29 @@ class S3SensorDecorator(FlowDecorator):
         "timeout_seconds": 3600,
```

Comment:
in above example: 

```python
    def formatter(path: str, flow_parameters: dict) -> str:
        path.format(year, flow_parameters["date"].split("-")[-1])
    @s3_sensor(
        path="s3://aip-example-sandbox/metaflow/S3SensorFlow/data/61/{flow_id}/year={year}",
        timeout_seconds=3600, # 1 hour
        polling_interval_seconds=90,
        path_formatter=formatter
    )
```

this doesn't work, year variable doesn't exist and would replace flow_id.
I suggest named format `path.format(variable=value, variable=value)`

Reply:
My bad, that was a syntax error. This will be updated to `path.format(year=flow_parameters["date"].split("-")[-1])`.

`flow_id` is a key in `flow_parameters` (not defined in the decorator) so it added automatically.